src/components/CardLists/planner.js

```diff
@@ -0,0 +1,32 @@
+import useStore from '../../hooks/useStore';
+import StyledCard from '../Card/styled';
+import StyledHeadline from '../CardHeadline/styled';
+
+import StyledCardlist from './styled';
+
+function CardlistPlanner() {
+	const cards = useStore(state => state.cards);
+	const deleteCard = useStore(state => state.deleteCard);
+
+	return (
+		<>
+			<p>Ausgewählte Übungen</p>
+			<StyledCardlist>
+				{cards.map(card => {
+					return (
+						<StyledCard
```

Comment:
If you have a list as parent element, you will need list items as children with the key attribute.